autoPyTorch/pipeline/components/setup/early_preprocessor/EarlyPreprocessing.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, random_state: Optional[np.random.RandomState] = None) -> None
         super().__init__()
```

Comment:
Do you know why those components files have capital letters in the file names?
Just asking, but I am not requiring the fixes in this PR.

Reply:
It relates to the functionality of [`find_components`](https://github.com/automl/Auto-PyTorch/blob/06e67de5017b4cccad9398e24a3d9f0bd8176da3/autoPyTorch/pipeline/components/base_component.py#L21)

Reply:
Thanks, it is also not a good convention, but let's stay as it is for now:(